source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/send_discord_post_to_telegram/send_discord_post_to_telegram.py

```diff
@@ -196,13 +209,30 @@ def _is_alphanumeric_with_dashes(string: str) -> bool:
 
     for telegram_chat, filter_ in telegram_chat_to_filter.items():
         if filter_(message=message):
-            await deps.telegram_bot_client.send_message(
-                entity=telegram_chat,
-                message=message_text,
-                parse_mode="md",
-                link_preview=False,
-                file=filename_urls or None,
-            )
+            if len(files) == 1:
+                # - Undownloaded videos and gifs can ONLY be send as single files
```

Comment:
be sent

Reply:
@marklidenberg Ты же знаешь что ты можешь быть милым и прислать сразу дифф чтобы я одну кнопку кликнул и оно проехало в код? :)
Вот так:
```suggestion
                # - Undownloaded videos and gifs can ONLY be sent as single files
```